.github/workflows/e2e-matrix-trigger-nap.yaml

```diff
@@ -0,0 +1,34 @@
+name: E2EMatrixTriggerNap
+on:
+  workflow_dispatch:
+    inputs:
+      location:
+        type: string
+        description: "the azure location to run the e2e test in"
+        default: "eastus"
+
+  workflow_run:
+    workflows: [ApprovalCommentNAP]
+    types: [completed]
+
+permissions:
+  contents: read
+
+jobs:
+  resolve:
+    if: github.event_name != 'workflow_run' || github.event.workflow_run.conclusion == 'success'
+    uses: ./.github/workflows/resolve-args.yaml
+  e2e-matrix:
+    permissions:
+      contents: read
+      id-token: write # This is required for requesting the JWT
+      statuses: write # ./.github/actions/commit-status/*          
+    needs: [resolve]
+    uses: ./.github/workflows/e2e-matrix-nap.yaml
```

Comment:
This file is so similar to the non-nap one that I think we could pass along a param on if its running in nap, or not